core/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Courses(models.Model):
```

Comment:
Courses -> Course

Reply:
Please use singular model names.

Reply:
Done. the model names are all singular now.

Reply:
It looks great. thank you.